<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-01-14T15:21:18 IST" name="Suite" finished-at="2022-01-14T15:21:29 IST" duration-ms="11003">
    <groups>
    </groups>
    <test started-at="2022-01-14T15:21:18 IST" name="EA Test" finished-at="2022-01-14T15:21:29 IST" duration-ms="11003">
      <class name="com.ea.tests.tests.HomePageTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.ea.tests.tests.HomePageTest@56620197]" started-at="2022-01-14T15:21:18 IST" name="setup" finished-at="2022-01-14T15:21:24 IST" duration-ms="6000" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLoginPageNavigation()[pri:0, instance:com.ea.tests.tests.HomePageTest@56620197]" started-at="2022-01-14T15:21:24 IST" name="verifyLoginPageNavigation" description="Verifies user is able to navigate to login page when user clicks on login link" finished-at="2022-01-14T15:21:27 IST" duration-ms="2491" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginPageNavigation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.ea.tests.tests.HomePageTest@56620197]" started-at="2022-01-14T15:21:27 IST" name="tearDown" finished-at="2022-01-14T15:21:28 IST" duration-ms="1117" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.ea.tests.tests.HomePageTest -->
      <class name="com.ea.tests.tests.LoginPageTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.ea.tests.tests.LoginPageTest@4fb0f2b9]" started-at="2022-01-14T15:21:18 IST" name="setup" finished-at="2022-01-14T15:21:24 IST" duration-ms="5998" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginTest()[pri:0, instance:com.ea.tests.tests.LoginPageTest@4fb0f2b9]" started-at="2022-01-14T15:21:24 IST" name="loginTest" description="Verifies user is able to login when user enters correct password and username and click on login button " finished-at="2022-01-14T15:21:28 IST" duration-ms="3595" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Home - Execute Automation Employee App] but found [Login - Execute Automation Employee App]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Home - Execute Automation Employee App] but found [Login - Execute Automation Employee App]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.ea.tests.tests.LoginPageTest.loginTest(LoginPageTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.ea.tests.tests.LoginPageTest@4fb0f2b9]" started-at="2022-01-14T15:21:28 IST" name="tearDown" finished-at="2022-01-14T15:21:29 IST" duration-ms="727" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.ea.tests.tests.LoginPageTest -->
    </test> <!-- EA Test -->
  </suite> <!-- Suite -->
</testng-results>
